identity/webapp/pages/account.tsx

```diff
@@ -239,9 +227,9 @@ const Profile: FC = () => {
                       rows={[
                         ['Title', 'Status', 'Pickup location'],
                         ...requests.results.map(result => [
-                          truncateTitle_REMOVE_THIS_FUNCTION_ASAP(
-                            result.item.title || result.workTitle || ''
-                          ),
+                          <TruncateTitle href={`/works/${result.workId}`}>
+                            {result.item.title || result.workTitle || ''}
```

Comment:
I know I did this originally, but looking at it, looks like there could potentially be a link without any text in it. Should this have some other placeholder text (e.g. 'unknown title') in those instances?